indigo_app/templates/indigo_api/_task_list_all.html

```diff
@@ -0,0 +1,55 @@
+{% load humanize indigo_app %}
+
+{% if tasks %}
+<table class="table table-striped table-bordered">
```

Comment:
The borders add too much noise, let's drop that and add `table-sm` to make it condensed -- we have a LOT of tasks.

Reply:
Done